### ISSUE TITLE
Make log format consistent across tools

### ISSUE THREAD

**Issue by cesarbs** [aspnet/DotNetTools#227]
E.g. when I try to use `sql-cache` or `user-secrets` with an invalid argument, the error format is different between the two:

**sql-cache**

```

$ dotnet sql-cache --foo
Specify --help for a list of available options and commands.
crit: Microsoft.Extensions.Caching.SqlConfig.Tools.Program[0]
      An error occurred. Unrecognized option '--foo'
```

**user-secrets**

```
$ dotnet user-secrets --foo
Specify --help for a list of available options and commands.
critical: Command failed : Unrecognized option '--foo'
```

`watch` errors have a different format too. For example:

```
[DotNetWatcher] fail: dotnet exit code: 134
```

**Comment by natemcmaster** [comment]
Also, we should make usage of colors consistent.

**Comment by natemcmaster** [comment]
Fixed in #237
